modules/dreamview/frontend/src/components/StatusBar/index.js

```diff
@@ -16,6 +18,11 @@ export default class StatusBar extends React.Component {
       showPlanningRSSInfo, monitor,
     } = this.props;
 
+    let electricityPercentage = meters.batteryPercentage;
```

Comment:
the batteryPercentage is set to null as default. Is there any reason why we need to do this change here? Also, if this really is needed, maybe set it in meters.js as it is the place for data manipulation.